src/command_modules/azure-cli-cdn/azure/cli/command_modules/cdn/_validators.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.util import CLIError
+
+
+def validate_origin(namespace):
+    def check_port_range(port, msg):
```

Comment:
I imagine the server would throw up an error if this were out of range, so you don't necessarily need to do this check.